xstream/nodes/sink_node.go

```diff
@@ -159,18 +159,22 @@ func (m *SinkNode) Open(ctx api.StreamContext, result chan<- error) {
 				var sink api.Sink
 				var err error
 				if !m.isMock {
+					logger.Debugf(fmt.Sprintf("Trying to get sink for rule %s with options %v\n", ctx.GetRuleId(), m.options))
```

Comment:
Debugf already allow format symbol. Do not need to nest fmt.Sprintf

Reply:
Updated